elastic/kibana
# [maps] hide create filter UI in canvas

Fixes https://github.com/elastic/kibana/issues/133802 and https://github.com/elastic/kibana/issues/133794

Hide create filter UI in canvas.

To test
* Install sample web log
* Create new canvas
* Import map into canvas
* Verify control to draw filters on map is not displayed. Verify controls to create filters from tooltips are not displayed
